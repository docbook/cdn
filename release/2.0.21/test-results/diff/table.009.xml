<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <p deltaxml:deltaV2="A">Nested table test.</p><div deltaxml:deltaV2="B" class="content">
            <p>Nested table test.</p>
            <p>CALS:</p>
            <figure id="R.1.4" class="informaltable-wrapper">
               <div class="informaltable">
                  <table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                     <colgroup>
                        <col class="tcol1" width="33%"/>
                        <col class="tcol2" width="33%"/>
                        <col class="tcol3" width="33%"/>
                     </colgroup>
                     <tbody>
                        <tr>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a foo</td>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a bar</td>
                           <td style="border-bottom: 1px solid ; ">a baz</td>
                        </tr>
                        <tr>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b foo</td>
                           <td>
                              <table border="0" width="100%" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                                 <colgroup>
                                    <col class="tcol1" width="37.04%"/>
                                    <col class="tcol2" width="31.48%"/>
                                    <col class="tcol3" width="31.48%"/>
                                 </colgroup>
                                 <tbody>
                                    <tr>
                                       <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' foo</td>
                                       <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' bar</td>
                                       <td style="border-bottom: 1px solid ; ">a' baz</td>
                                    </tr>
                                    <tr>
                                       <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' foo</td>
                                       <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' bar</td>
                                       <td style="border-bottom: 1px solid ; " rowspan="2">b' and c' baz</td>
                                    </tr>
                                    <tr>
                                       <td style="border-right: 1px solid ; ">c' foo</td>
                                       <td style="border-right: 1px solid ; ">c' bar</td>
                                    </tr>
                                 </tbody>
                              </table>
                           </td>
                           <td style="border-bottom: 1px solid ; ">b baz</td>
                        </tr>
                        <tr>
                           <td style="border-right: 1px solid ; ">c foo</td>
                           <td style="border-right: 1px solid ; ">c bar</td>
                           <td>c baz</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </figure>
            <p>Nested table test.</p>
            <figure id="R.1.6" class="informaltable-wrapper">
               <div class="informaltable">
                  <table border="0" style="border-collapse: collapse;">
                     <colgroup>
                        <col class="tcol1" width="33%"/>
                        <col class="tcol2" width="33%"/>
                        <col class="tcol3" width="33%"/>
                     </colgroup>
                     <tbody>
                        <tr>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a foo</td>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a bar</td>
                           <td style="border-bottom: 1px solid ; ">a baz</td>
                        </tr>
                        <tr>
                           <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b foo</td>
                           <td>
                              <table border="0" width="100%" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                                 <colgroup>
                                    <col class="tcol1" width="37.04%"/>
                                    <col class="tcol2" width="31.48%"/>
                                    <col class="tcol3" width="31.48%"/>
                                 </colgroup>
                                 <tbody>
                                    <tr>
                                       <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' foo</td>
                                       <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' bar</td>
                                       <td style="border-bottom: 1px solid ; ">a' baz</td>
                                    </tr>
                                    <tr>
                                       <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' foo</td>
                                       <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' bar</td>
                                       <td style="border-bottom: 1px solid ; " rowspan="2">b' and c' baz</td>
                                    </tr>
                                    <tr>
                                       <td style="border-right: 1px solid ; ">c' foo</td>
                                       <td style="border-right: 1px solid ; ">c' bar</td>
                                    </tr>
                                 </tbody>
                              </table>
                           </td>
                           <td style="border-bottom: 1px solid ; ">b baz</td>
                        </tr>
                        <tr>
                           <td style="border-right: 1px solid ; ">c foo</td>
                           <td style="border-right: 1px solid ; ">c bar</td>
                           <td>c baz</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </figure>
            <p>HTML:</p>
            <figure id="R.1.8" class="informaltable-wrapper">
               <div class="informaltable">
                  <table border="1">
                     <tbody>
                        <tr>
                           <td>a foo</td>
                           <td>a bar</td>
                           <td>a baz</td>
                        </tr>
                        <tr>
                           <td>b foo</td>
                           <td>
                              <table width="300">
                                 <tbody>
                                    <tr>
                                       <td>a' foo</td>
                                       <td>a' bar</td>
                                       <td>a' baz</td>
                                    </tr>
                                    <tr>
                                       <td>b' foo</td>
                                       <td>b' bar</td>
                                       <td rowspan="2">b' and c' baz</td>
                                    </tr>
                                    <tr>
                                       <td>c' foo</td>
                                       <td>c' bar</td>
                                    </tr>
                                 </tbody>
                              </table>
                           </td>
                           <td>b baz</td>
                        </tr>
                        <tr>
                           <td>c foo</td>
                           <td>c bar</td>
                           <td>c baz</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </figure>
            <p>Nested table test.</p>
            <figure id="R.1.10" class="informaltable-wrapper">
               <div class="informaltable">
                  <table>
                     <tbody>
                        <tr>
                           <td>a foo</td>
                           <td>a bar</td>
                           <td>a baz</td>
                        </tr>
                        <tr>
                           <td>b foo</td>
                           <td>
                              <table width="200">
                                 <tbody>
                                    <tr>
                                       <td>a' foo</td>
                                       <td>a' bar</td>
                                       <td>a' baz</td>
                                    </tr>
                                    <tr>
                                       <td>b' foo</td>
                                       <td>b' bar</td>
                                       <td rowspan="2">b' and c' baz</td>
                                    </tr>
                                    <tr>
                                       <td>c' foo</td>
                                       <td>c' bar</td>
                                    </tr>
                                 </tbody>
                              </table>
                           </td>
                           <td>b baz</td>
                        </tr>
                        <tr>
                           <td>c foo</td>
                           <td>c bar</td>
                           <td>c baz</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </figure>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">CALS:</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="R.1.4" class="informaltable-wrapper">
            <div class="informaltable">
               <table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                  <colgroup>
                     <col class="tcol1" width="33%"/>
                     <col class="tcol2" width="33%"/>
                     <col class="tcol3" width="33%"/>
                  </colgroup>
                  <tbody>
                     <tr>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a foo</td>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a bar</td>
                        <td style="border-bottom: 1px solid ; ">a baz</td>
                     </tr>
                     <tr>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b foo</td>
                        <td>
                           <table border="0" width="100%" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                              <colgroup>
                                 <col class="tcol1" width="37.04%"/>
                                 <col class="tcol2" width="31.48%"/>
                                 <col class="tcol3" width="31.48%"/>
                              </colgroup>
                              <tbody>
                                 <tr>
                                    <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' foo</td>
                                    <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' bar</td>
                                    <td style="border-bottom: 1px solid ; ">a' baz</td>
                                 </tr>
                                 <tr>
                                    <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' foo</td>
                                    <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' bar</td>
                                    <td style="border-bottom: 1px solid ; " rowspan="2">b' and c' baz</td>
                                 </tr>
                                 <tr>
                                    <td style="border-right: 1px solid ; ">c' foo</td>
                                    <td style="border-right: 1px solid ; ">c' bar</td>
                                 </tr>
                              </tbody>
                           </table>
                        </td>
                        <td style="border-bottom: 1px solid ; ">b baz</td>
                     </tr>
                     <tr>
                        <td style="border-right: 1px solid ; ">c foo</td>
                        <td style="border-right: 1px solid ; ">c bar</td>
                        <td>c baz</td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </figure><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">Nested table test.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="R.1.6" class="informaltable-wrapper">
            <div class="informaltable">
               <table border="0" style="border-collapse: collapse;">
                  <colgroup>
                     <col class="tcol1" width="33%"/>
                     <col class="tcol2" width="33%"/>
                     <col class="tcol3" width="33%"/>
                  </colgroup>
                  <tbody>
                     <tr>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a foo</td>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a bar</td>
                        <td style="border-bottom: 1px solid ; ">a baz</td>
                     </tr>
                     <tr>
                        <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b foo</td>
                        <td>
                           <table border="0" width="100%" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                              <colgroup>
                                 <col class="tcol1" width="37.04%"/>
                                 <col class="tcol2" width="31.48%"/>
                                 <col class="tcol3" width="31.48%"/>
                              </colgroup>
                              <tbody>
                                 <tr>
                                    <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' foo</td>
                                    <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' bar</td>
                                    <td style="border-bottom: 1px solid ; ">a' baz</td>
                                 </tr>
                                 <tr>
                                    <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' foo</td>
                                    <td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' bar</td>
                                    <td style="border-bottom: 1px solid ; " rowspan="2">b' and c' baz</td>
                                 </tr>
                                 <tr>
                                    <td style="border-right: 1px solid ; ">c' foo</td>
                                    <td style="border-right: 1px solid ; ">c' bar</td>
                                 </tr>
                              </tbody>
                           </table>
                        </td>
                        <td style="border-bottom: 1px solid ; ">b baz</td>
                     </tr>
                     <tr>
                        <td style="border-right: 1px solid ; ">c foo</td>
                        <td style="border-right: 1px solid ; ">c bar</td>
                        <td>c baz</td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </figure><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">HTML:</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="R.1.8" class="informaltable-wrapper">
            <div class="informaltable">
               <table border="1">
                  <tbody>
                     <tr>
                        <td>a foo</td>
                        <td>a bar</td>
                        <td>a baz</td>
                     </tr>
                     <tr>
                        <td>b foo</td>
                        <td>
                           <table width="300">
                              <tbody>
                                 <tr>
                                    <td>a' foo</td>
                                    <td>a' bar</td>
                                    <td>a' baz</td>
                                 </tr>
                                 <tr>
                                    <td>b' foo</td>
                                    <td>b' bar</td>
                                    <td rowspan="2">b' and c' baz</td>
                                 </tr>
                                 <tr>
                                    <td>c' foo</td>
                                    <td>c' bar</td>
                                 </tr>
                              </tbody>
                           </table>
                        </td>
                        <td>b baz</td>
                     </tr>
                     <tr>
                        <td>c foo</td>
                        <td>c bar</td>
                        <td>c baz</td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </figure><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">Nested table test.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="R.1.10" class="informaltable-wrapper">
            <div class="informaltable">
               <table>
                  <tbody>
                     <tr>
                        <td>a foo</td>
                        <td>a bar</td>
                        <td>a baz</td>
                     </tr>
                     <tr>
                        <td>b foo</td>
                        <td>
                           <table width="200">
                              <tbody>
                                 <tr>
                                    <td>a' foo</td>
                                    <td>a' bar</td>
                                    <td>a' baz</td>
                                 </tr>
                                 <tr>
                                    <td>b' foo</td>
                                    <td>b' bar</td>
                                    <td rowspan="2">b' and c' baz</td>
                                 </tr>
                                 <tr>
                                    <td>c' foo</td>
                                    <td>c' bar</td>
                                 </tr>
                              </tbody>
                           </table>
                        </td>
                        <td>b baz</td>
                     </tr>
                     <tr>
                        <td>c foo</td>
                        <td>c bar</td>
                        <td>c baz</td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </figure>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>