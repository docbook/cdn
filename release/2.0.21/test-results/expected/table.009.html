<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>Nested table test.</p><p>CALS:</p><figure id="R.1.4" class="informaltable-wrapper"><div class="informaltable"><table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="33%" /><col class="tcol2" width="33%" /><col class="tcol3" width="33%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a foo</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a bar</td><td style="border-bottom: 1px solid ; ">a baz</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b foo</td><td><table border="0" width="100%" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="37.04%" /><col class="tcol2" width="31.48%" /><col class="tcol3" width="31.48%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' foo</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' bar</td><td style="border-bottom: 1px solid ; ">a' baz</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' foo</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' bar</td><td style="border-bottom: 1px solid ; " rowspan="2">b' and c' baz</td></tr><tr><td style="border-right: 1px solid ; ">c' foo</td><td style="border-right: 1px solid ; ">c' bar</td></tr></tbody></table></td><td style="border-bottom: 1px solid ; ">b baz</td></tr><tr><td style="border-right: 1px solid ; ">c foo</td><td style="border-right: 1px solid ; ">c bar</td><td>c baz</td></tr></tbody></table></div></figure><p>Nested table test.</p><figure id="R.1.6" class="informaltable-wrapper"><div class="informaltable"><table border="0" style="border-collapse: collapse;"><colgroup><col class="tcol1" width="33%" /><col class="tcol2" width="33%" /><col class="tcol3" width="33%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a foo</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a bar</td><td style="border-bottom: 1px solid ; ">a baz</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b foo</td><td><table border="0" width="100%" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="37.04%" /><col class="tcol2" width="31.48%" /><col class="tcol3" width="31.48%" /></colgroup><tbody><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' foo</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">a' bar</td><td style="border-bottom: 1px solid ; ">a' baz</td></tr><tr><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' foo</td><td style="border-right: 1px solid ; border-bottom: 1px solid ; ">b' bar</td><td style="border-bottom: 1px solid ; " rowspan="2">b' and c' baz</td></tr><tr><td style="border-right: 1px solid ; ">c' foo</td><td style="border-right: 1px solid ; ">c' bar</td></tr></tbody></table></td><td style="border-bottom: 1px solid ; ">b baz</td></tr><tr><td style="border-right: 1px solid ; ">c foo</td><td style="border-right: 1px solid ; ">c bar</td><td>c baz</td></tr></tbody></table></div></figure><p>HTML:</p><figure id="R.1.8" class="informaltable-wrapper"><div class="informaltable"><table border="1"><tbody><tr>
<td>a foo</td>
<td>a bar</td>
<td>a baz</td>
</tr><tr>
<td>b foo</td>
<td>
<table width="300"><tbody><tr>
      <td>a' foo</td>
      <td>a' bar</td>
      <td>a' baz</td>
    </tr><tr>
      <td>b' foo</td>
      <td>b' bar</td>
      <td rowspan="2">b' and c' baz</td>
    </tr><tr>
      <td>c' foo</td>
      <td>c' bar</td>
    </tr></tbody></table>
</td>
<td>b baz</td>
</tr><tr>
<td>c foo</td>
<td>c bar</td>
<td>c baz</td>
</tr></tbody></table></div></figure><p>Nested table test.</p><figure id="R.1.10" class="informaltable-wrapper"><div class="informaltable"><table><tbody><tr>
<td>a foo</td>
<td>a bar</td>
<td>a baz</td>
</tr><tr>
<td>b foo</td>
<td>
<table width="200"><tbody><tr>
      <td>a' foo</td>
      <td>a' bar</td>
      <td>a' baz</td>
    </tr><tr>
      <td>b' foo</td>
      <td>b' bar</td>
      <td rowspan="2">b' and c' baz</td>
    </tr><tr>
      <td>c' foo</td>
      <td>c' bar</td>
    </tr></tbody></table>
</td>
<td>b baz</td>
</tr><tr>
<td>c foo</td>
<td>c bar</td>
<td>c baz</td>
</tr></tbody></table></div></figure></article><script src="../../resources/js/prism.js"></script></body></html>